src/internal/components/LayoutEditor.tsx

```diff
@@ -97,7 +110,7 @@ export const LayoutEditor = (props: LayoutEditorProps) => {
    *  - if no match, use saveState directly and clear localStorage
    */
   const loadPuckInitialHistory = useCallback(() => {
-    if (!templateMetadata) {
+    if (!templateMetadata || localDev) {
```

Comment:
I don't think we need to prevent the localStorage stuff, but we would want to stop the real saveState from trying to send.